src/topp/MzTabExporter.cpp

```diff
@@ -730,7 +730,7 @@ namespace OpenMS
 
       // currently we don't save how many channels we saved in a consensus feature so we scan the file
       // for the maximum max index in a consensus feature and take this as number of study variables
```

Comment:
what is
https://github.com/timosachsenberg/OpenMS/blob/fix/MzTab_column_count/src/topp/MzTabExporter.cpp#L834
doing?